backend/copr_backend/msgbus.py

```diff
@@ -33,6 +39,8 @@ def message_from_worker_job(style, topic, job, who, ip, pid):
     once we can exepect that people are using copr-messaging module (and thus
     we can change message format without affecting the API).
     """
+    if not copr_messaging_enabled:
+        return None
```

Comment:
It looks like this is not needed, what is the traceback you face if you uninstall python-copr-messaging?

MessageSender.announce() calls `bus.announce_job()` only if some bus is configured (which shouldn't be your case) therefore neither `message_from_worker_job` shouldn't be called.

Reply:
Yes, in today's codebase, there's no need to add these lines.
But i want to make this method more robust, maybe it was call by another function in future, and the caller may not know our assumptions.
The `schema` in the context is undefined and may cause an exception.

Reply:
This just feels weird, maybe the position in code, I'm not sure.

If things work for you just like that, can we just (and resolve the rest later?):
```
ry:
    from copr_messaging import schema
except ImportError:
    # copr_messaging is optional
    schema = None
```